platform/src/main/java/org/stellar/anchor/platform/config/PropertySep10Config.java

```diff
@@ -87,6 +87,18 @@ void validateConfig(Errors errors) {
       errors.rejectValue(
           "homeDomain", "home-domain-empty", "The sep10.home_domain is not defined.");
     } else {
+      try {
+        new ManageDataOperation.Builder(String.format("%s %s", homeDomain, "auth"), new byte[64])
```

Comment:
Can we make the length check more explicit? Something like:
```java
byte[] bytes = homeDomain.getBytes(StandardCharsets.US_ASCII);
if (bytes.length > 64) {
     errors.reject( ... )
}
```

Reply:
That's a good alternative. However, the size check (<=64) happens in the `ManageDataOperation` constructor. It would be more robust if we call the `ManageDataOperation`.

Reply:
I'm not sure if I'm convinced that is more robust. For example, if there are new validations added to the `ManageDataOperation` constructor, we will fail with a message complaining about the size. Or if the allowed since increases from 64, we will need to update the message here.

Reply:
This is actually intended. If there is a new validation added to `ManageDataOperation`, the validation should fail because AP uses `ManageDataOperation` to create SEP-10 challenge.